app/views/content/train-to-be-a-teacher/get-school-experience.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Get school experience"
+heading: "Get school experience"
+description: |-
+  Get school experience to explore what life is like in the classroom before you start your initial teacher training (ITT). Discover if teaching is for you.
+related_content:
+    Who do you want to teach? : "/train-to-be-a-teacher/who-do-you-want-to-teach"
+    School experience helped me decide to switch : "/my-story-into-teaching/career-changers/school-experience-helped-me-decide-to-switch"
+    Train to teach events : "/event-categories/train-to-teach-events"
```

Comment:
```suggestion
    Train to Teach events : "/event-categories/train-to-teach-events"
```

@EllieNodder Just flagging this - we need to capitalise 'Teach' wherever referenced as it is the brand name of their events